.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,4 +17,4 @@
 - ##### Reviewer:
 
 - [ ] Reviewed Tests
-
+- [ ] Has good error handling and messaging
```

Comment:
I don't think we can add this yet, particularly if "messaging" means "logging". 

The user story itself should call out how error scenarios should be handled. Thus, if the tests are sufficient for a story, then that means it is addressing the error scenarios. But if those scenarios are not defined in the story, then it's very subjective as to what "good error handling" means.